src/libertem/executor/delayed.py

```diff
@@ -121,6 +121,13 @@ def get_available_workers(self):
             Worker(name='delayed', host='localhost', resources=resources)
         ])
 
+    @classmethod
+    def get_resources(cls, udfs, user_backends=tuple()):
```

Comment:
This duplicates the function on line 171. Happy to use your and version/naming, though.

Reply:
Ah, didn't see it. Gone it is! :-)